src/tool_manager.ts

```diff
@@ -197,7 +197,7 @@ class ToolManager {
       vscode.window.showErrorMessage("Invalid editor path to run the project");
     } else {
       let terminal = vscode.window.createTerminal("Godot");
-      let cmmand = `${editorPath.replace(" ", "\\ ")} ${params}`;
+      let cmmand = `${editorPath.replace(" ", "` ")} ${params}`;
```

Comment:
This is the character for a strong literal in Windows PowerShell, which is what vscode uses internally on Windows platforms to launch the terminal commands.  It doesn't format properly when using `\` and thus the commands fail to execute on Windows when invoked.

Reply:
Sorry, string literal.  The grave accent character is what replaces spaces into their literal form so that paths are considered as a single argument by the shell on Windows.